### DISCUSSION TITLE
Comparing Web app and Python lib for visualizing color gamuts

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#879] [Q&A] [answered]
Hi Colour Team,

I use both colour-science web app and Python library to visualize Color Gamut (sRGB and ProPhoto in CIE Lab).
But they are quite different.
![image](https://user-images.githubusercontent.com/16201681/134756197-64838d34-f2a7-49f4-88ba-af6c6a051618.png)

Here is my Python code:
```python
prophotorgb = colour.models.PROPHOTO_RGB_COLOURSPACE
srgb = colour.models.sRGB_COLOURSPACE

colour.plotting.plot_RGB_colourspaces_gamuts(
    (srgb, prophotorgb),
    reference_colourspace='CIE Lab',
    chromatically_adapt=True,
    face_colours=(None, (0.25, 0.25, 0.25)),
    edge_colours=(None, (0.25, 0.25, 0.25)),
    edge_alpha=(1.0, 0.1),
    face_alpha=(1.0, 0.0))
```

Did I make any mistake?

Thanks in advance.

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

So they look the same to me, except that one has an axis that is swap! I will have to check which one is correct. The other shape/volume differences are because the web app uses perspective vs orthographic projection for MPL.

Cheers,

Thomas

**Reply by thomasmansencal** [reply]
Looking at it quickly, I think that the Web App axes are not properly reordered. I need to update it anyway so I will stash that on the TODO list!

**Reply by hminle** [reply]
Hi @KelSolaar 

> The other shape/volume differences are because the web app uses perspective vs orthographic projection for MPL.

Could you elaborate more about this?
What is MPL?
I want to make the shape/volume in Python similar to the web app.

For me, I think the axes from both the webapp and Python are the same. Not sure what you look at.

**Reply by hminle** [reply]
The shapes in web app are similar to Color Sync in Mac
![image](https://user-images.githubusercontent.com/16201681/135154515-68e79307-263f-48bd-9562-8734015bff8f.png)

I want to make the Python visualize the same shape

**Reply by thomasmansencal** [reply]
Hello,

If you look at the axes labels, the `a` and `b` are swapped in the Web App.

There is unfortunately no way to get Matplotlib render 3D plots with a perspective projection, it is orthographic all the way! :(

Cheers,

Thomas

**Reply by hminle** [reply]
Thank you Thomas, I got it. :)
